packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.test.tsx

```diff
@@ -918,6 +919,48 @@ describe('ContextualMenu', () => {
     expect(customRenderer).toHaveBeenCalledTimes(2);
   });
 
+  describe('ContextualMenu snapshot', () => {
+    it('ContextualMenu should be present in DOM when hidden', () => {
```

Comment:
Suggestion: I see that this test has the same name as the one from line 1004. Is there some differentiation we can make in the naming?